overview.tt

```diff
@@ -0,0 +1,279 @@
+[% WRAPPER layout.tt title="Overview" handlesLayout=1 %]
+
+<div class="page-title">
+  <h1>Overview</h1>
+</div>
+
+<section class="overview-description">
+  Step on a journey to explore and learn why so many developers, system
```

Comment:
```suggestion
  Find out why so many developers, system
```

I think "step on a journey" is not really idiomatic. Maybe "Begin your journey" but in any case a journey sounds like a lot of work...